book/installation.rst

```diff
@@ -126,6 +126,11 @@ Read the :doc:`Symfony Release process </contributing/community/releases>`
 to better understand why there are several Symfony versions and which one
 to use for your projects.
 
+.. tip::
+
+    Note that if you install a specific version, you will want to make sure 
+    you refer to the same version of the documentation.
```

Comment:
Maybe replace this with "[...] make sure to read the corresponding version of the documentation." What do you think?